adafruit_pcf8591/analog_in.py

```diff
@@ -27,11 +27,16 @@
 * Author(s): Bryan Siepert, adpted from ADS1x15 by Carter Nelson
 """
 
+try:
+    import typing  # pylint: disable=unused-import
+except ImportError:
+    pass
+
 
 class AnalogIn:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, pin):
+    def __init__(self, pcf: "PCF8591", pin: int) -> None:
```

Comment:
Would you mind fixing the docstring parameter description typing to be `int` to match?  Also, it says `ads` for the other, but should be `pcf`.  Thanks!

Reply:
wow, I saw the int issue after I tracked it down.  meant to put a question in comment here to ask about fixing it, totally missed the ads.  Makes sense since this was adapted from that driver....